src/generated/resources/assets/gtceu/lang/en_us.json

```diff
@@ -1922,7 +1917,6 @@
   "compass.node.gtceu.multiblock/multiblock_tank": "Multiblock Tank",
   "compass.node.gtceu.multiblock/primitive_blast_furnace": "Primitive Blast Furnace",
   "compass.node.gtceu.multiblock/primitive_pump": "Primitive Pump",
-  "compass.node.gtceu.multiblock/processing_array": "Processing Array",
```

Comment:
You need to enable the`doProcessingArray` and `generateLowQualityGems` configs.

Reply:
fixed